25.1.3+ (#3)
DevExpress-Examples/reporting-use-devexpress-drawing-skia-engine: Readme.md

Reviewer: ```suggestion
You can run the app on the Windows platform, the Windows Subsystem for Linux, or Docker. To launch the app with Docker, select _Docker_ from the Launch drop-down menu in the Visual Studio toolbar.
```

Reviewer: ```suggestion
Run the application using the .NET CLI on Windows, Linux, or macOS with the following command:
```

> ## How to Build and Run this Example

Reviewer: ```suggestion
## How to Build and Run This Example
```

### Obtain Your Personal DevExpress License Key

> Building a project within a dockerized environment cannot obtain the host's key automatically - which is usually supplied as a build-time secret. Refer to [License Key for DevExpress .NET Products](https://docs.devexpress.com/GeneralInformation/405494/trial-register/set-up-your-dev-express-license-key).

Reviewer: ```suggestion
Building a project within a dockerized environment does not obtain the host's key automatically - which is usually supplied as a build-time secret. Refer to [License Key for DevExpress .NET Products](https://docs.devexpress.com/GeneralInformation/405494/trial-register/set-up-your-dev-express-license-key) for details.
```
